content/en/docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -71,15 +71,15 @@ gate for each individual option.
 
 ### Changing the CPU Manager Policy
 
-Since the CPU manger policy can only be applied when kubelet spawns new pods, simply changing from
-"none" to "static" won't apply to existing pods. So in order to properly change the CPU manager
+Since the CPU manger policy can only be applied when kubelet spawns new pods, changing from
```

Comment:
i think it's a typo here shouldn't this be `Since the CPU manager`?

Reply:
Yes, that should be `manager`

Reply:
I don't see your commits for this change. Could you add the commit here .
Thanks!

Reply:
Added. Thanks!